cypress/integration/entitybrowser_spec.js

```diff
@@ -11,62 +11,80 @@ describe('Entity Browser', function() {
-      cy.get('[data-cy=cypress-form-field]')
-        .should('have.length.above', 1)
+      cy.get('[data-cy=cypress-form-field]').then(el => {
+        cy.wrap(el).should('have.length.above', 1)
+      })
     })
 
     it('should change value in detail view and save', function() {
-      cy.get('input#input-detailForm-callname')
-        .type('{selectall}{del}Test Callname')
-        .should('have.value', 'Test Callname')
+      cy.get('input#input-detailForm-callname').then(el => {
+        cy.wrap(el)
+          .type('{selectall}{del}Test Callname')
+          .should('have.value', 'Test Callname')
+      })
       cy.get('[data-cy=detail-form_submit-button]').children().first()
         .click()
-      cy.contains('Zuletzt gespeichert')
+      cy.contains('Zuletzt gespeichert', {timeout: 4000})
     })
 
     it('should change page back to list view without warning', function() {
-      cy.get('[data-cy=entity-detail_back-button]').children().first().children().first()
-        .click()
-      cy.get('[data-list-cell]').should('have.length.above', 1)
+      cy.get('[data-cy=entity-detail_back-button]').children().first().then(el => {
+        cy.wrap(el).click()
+      })
+      cy.get('[data-cy=list-cell]').then(el => {
+        const listCells = el.children()
+        cy.wrap(listCells).should('have.length.above', 10)
+      })
     })
 
     it('should display warning on changing page back to list view on edited form', function() {
       cy.get('input#input-detailForm-callname')
         .type('{selectall}{del}')
-      cy.get('[data-cy=entity-detail_back-button]').children().first().children().first()
-        .click()
-      cy.contains('ungespeicherte Änderungen')
+      cy.get('[data-cy=entity-detail_back-button]').children().first().then(el => {
+        cy.wrap(el).click()
+      })
+      cy.contains('ungespeicherte Änderungen', {timeout: 2000})
     })
   })
 
   describe('SearchForm', () => {
     it('should display extended search form', function() {
-      cy.get('[data-cy=extend-search-button]').children()
-        .click()
-      cy.get('[data-cy=cypress-form-field]')
-        .should('have.length.above', 1)
+      cy.get('[data-cy=extend-search-button]').children().first()
+        .click({force: true})
+      cy.get('[data-cy=cypress-form-field]').then(el => {
+        cy.wrap(el).should('have.length.above', 10)
+      })
     })
 
     it('should reset list view through search button reset', function() {
       cy.get('[data-cy=cypress-form-field]').children().children()
         .type('Test Person')
-      cy.get('[data-cy=search-form_reset-button]').next().children().first()
-        .click()
-      cy.get('[data-list-cell]').should('have.length.above', 1)
+      cy.get('[data-cy=search-form_reset-button]')
+        .click({force: true})
+      cy.get('[data-cy=list-cell]').then(el => {
+        const listCells = el.children()
+        cy.wrap(listCells).should('have.length.above', 10)
```

Comment:
does this really verify what you want to test?
Shouldn't you check if "Test PersoN" isnt in the input field after clicking result rather than check if there are 10 record?